Configuration/TCA/tx_ats_domain_model_job.php

```diff
@@ -566,5 +566,11 @@
                 'size' => 30,
             ],
         ],
+        'media' => [
+            'exclude' => 1,
+            'label' => 'LLL:EXT:ats/Resources/Private/Language/locallang_db.xlf:tx_ats_domain_model_job.media',
+            'config' => \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::getFileFieldTCAConfig('media', [
+            ], $GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext']),
```

Comment:
Are you sure that the default makes sense here? For the reference this is the current list in TYPO3v8:

    gif,jpg,jpeg,tif,tiff,bmp,pcx,tga,png,pdf,ai,svg

I see quite a few formats which would be unusual in jobs. ;-)

A hardcoded custom list would be safer here IMO.

Reply:
Changed, thank you.